doc/api/tls.md

```diff
@@ -1406,7 +1406,9 @@ to `true`, other APIs that create secure contexts leave it unset.
 from `process.argv` as the default value of the `sessionIdContext` option, other
 APIs that create secure contexts have no default value.
 
-The `tls.createSecureContext()` method creates a credentials object.
+The `tls.createSecureContext()` method creates a `SecureContext` object. The
+object has no public methods, but is accepted as an argument to several `tls`
+APIs, such as [`tls.createServer()`][] and [`server.addContext()`][].
```

Comment:
Missing bottom reference for `` [`server.addContext()`][] ``.

Reply:
Sorry, I thought `make test-doc` would check for that.

Reply:
IIRC, it checks for erroneous #hashes and orphaned bottom references, while these cases are still unchecked.